RFS/src/main/java/com/rfs/common/DocumentReindexer.java

```diff
@@ -30,7 +30,13 @@ public Mono<Void> reindex(
         Flux<Document> documentStream,
         IDocumentMigrationContexts.IDocumentReindexContext context
     ) {
+        // Create elastic scheduler for long-lived i/o bound tasks
```

Comment:
Could you write a test case that confirms the differences in processing speeds for the flux source?

Reply:
What are you envisioning for this? Is this a blocking comment given the test I added for parallelization checks?

Reply:
The new test cases you add cover this scenario - thanks